mantidproject/mantid
# Add Hinting line edit to options column

**Description of work.**

This adds the line edit hints to the options column of the SANS table. It has had the branch from ~~~#23362~~~ merged in so should be merged after that branch has been merged into master and then had master merged into the release branch

**To test:**

* Open the SANS GUI
* Type 'W' into the options column you should be shown a hint for WavelengthMin and WavelengthMax
* Type 'E' into the options column you should be shown a hint for EventSlices

Fixes #23022 . 



*This does not require release notes* because being merged into feature branch

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
